AsyncDisplayKit/Details/ASFlowLayoutController.mm

```diff
@@ -105,8 +105,8 @@ - (NSSet *)workingRangeIndexPathsForScrolling:(enum ASScrollDirection)scrollDire
   if (_layoutDirection == ASFlowLayoutDirectionHorizontal) {
     ASDisplayNodeAssert(scrollDirection == ASScrollDirectionNone || scrollDirection == ASScrollDirectionLeft || scrollDirection == ASScrollDirectionRight, @"Invalid scroll direction");
 
-    CGFloat backScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.leadingBufferScreenfuls : self.tuningParameters.trailingBufferScreenfuls;
-    CGFloat frontScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.trailingBufferScreenfuls : self.tuningParameters.leadingBufferScreenfuls;
+    int backScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.leadingBufferScreenfuls : self.tuningParameters.trailingBufferScreenfuls;
```

Comment:
this is a CGFloat as of 38f8e63b01e8a7565d854b49b54e47459c98d0b8, no change necessary =]